test/libsolidity/GasTest.cpp

```diff
@@ -128,14 +128,14 @@ TestCase::TestResult GasTest::run(std::ostream& _stream, std::string const& _lin
 		return TestResult::FatalError;
 	}
 
-	Json::Value estimateGroups = compiler().gasEstimates(compiler().lastContractName());
+	Json estimateGroups = compiler().gasEstimates(compiler().lastContractName());
 	if (
 		m_expectations.size() == estimateGroups.size() &&
 		boost::all(m_expectations, [&](auto const& expectations) {
 		auto const& estimates = estimateGroups[expectations.first];
 		return estimates.size() == expectations.second.size() &&
 			boost::all(expectations.second, [&](auto const& entry) {
-				return entry.second == estimates[entry.first].asString();
+				return entry.second == estimates[entry.first].template get<std::string>();
```

Comment:
This is the only place in which you use the weird ``.template`` syntax - did that become obsolete with C++17? I don't quite remember. But probably also redundant here, since it works without everywhere else.

Reply:
YEah I was also quite surprised that this is still needed, but it only works with that.

```
~/git/solidity/test/libsolidity/GasTest.cpp:138:51: error: use 'template' keyword to treat 'get' as a dependent template name
                                return entry.second == estimates[entry.first].get<std::string>();
                                                                              ^
                                                                              template 
```

Reply:
Strange... but alright, doesn't hurt either